Source/Csla/DataPortalClient/HttpProxy.cs

```diff
@@ -117,13 +117,22 @@ protected override async Task<byte[]> CallDataPortalServer(byte[] serialized, st
     protected virtual void SetHttpRequestHeaders(HttpRequestMessage request)
     { }
 
+    /// <summary>
+    /// Override to set headers or other properties of the
+    /// WebClient before it is sent to the server.
+    /// </summary>
```

Comment:
Maybe document here that this is _only_ used for the sync-CSLA-method versions and clarify in `SetHttpRequestHeader` that it's _only_ used for the async-versions?